updateFromCurator.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -xeou pipefail
+
+# updateFromCurator.sh allows automated hash updating.
+# It is intended to be used in the curatorbin root dir.
+# VERSION_UPDATE will increment the major/minor/patch versions as specified
```

Comment:
Can you give examples of using the `${VERSION_UPDATE}` syntax? It seems like that doesn't work and you can only pass which particular part of the version string you can pass.

Can you enumerate the allowed increment strings (major, minor, patch)?